java/client/src/org/openqa/selenium/remote/internal/UserInfo.java

```diff
@@ -0,0 +1,29 @@
+package org.openqa.selenium.remote.internal;
+
+public final class UserInfo
```

Comment:
We can reformat this code, but it's unusual for it not to follow the project's style.